ci-operator/step-registry/cucushift/installer/rehearse/gcp/upi/disconnected-proxy/cucushift-installer-rehearse-gcp-upi-disconnected-proxy-workflow.metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+	"path": "cucushift/installer/rehearse/gcp/upi/disconnected-proxy/cucushift-installer-rehearse-gcp-upi-disconnected-proxy-workflow.yaml",
```

Comment:
Can we replace `disconnected-proxy` with `proxy` here, btw, I saw ipi also using the same folder name, if possible, can we update them together in this PR?

Reply:
So you think we don't need the word "disconnected" to indicate it's a disconnected/restricted network?

Reply:
When we are testing cluster behind proxy to validate all the outgoing traffic is via proxy, we have to make the network is restricted/disconnected, so that means restrict/disconnected is a requirement for testing proxy. So I think we can remove the word "disconnected".